### DISCUSSION TITLE
Cambio Titolo Barra Laterale

### DISCUSSION THREAD

**Discussion by vinny74man** [indomus/forum#1609] [HUB: Home Assistant]
Ciao a tutti vorrei sapere se fosse possibile personalizzare il titolo della barra laterale di home assistant,, in ogni pagina tenendo visibile la barra laterale il titolo è "Home Assistant" io vorrei personalizzarlo, c'è modo di farlo?
Grazie

**Comment by 50494554524F** [comment]
estensione "custom sidebar", la trovi in hacs

**Comment by vinny74man** [comment]
Grazie

Scarica Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
